projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -138,15 +125,17 @@ export async function downloadJobRequest() {
  * The information about the data job is retrieved from jobData object and sent as JSON.
  */
 export async function createJobRequest() {
```

Comment:
Consider abstracting out the `doSomethingJobRequest` methods to single method. They don't seem that different.

Reply:
done